src/includes/utils/stoi.hpp

```diff
@@ -0,0 +1,11 @@
+#ifndef STOI_HPP
+#define STOI_HPP
+
+# include <string>
+# include "isDigit.hpp"
+
+namespace utils {
+	size_t		stoi(const std::string& str);
+}
+
+#endif //STOI_HPP
```

Comment:
Remove file

Reply:
it was itos that had to be removed right?